com.unity.render-pipelines.universal/Runtime/Data/UniversalRenderPipelineAsset.cs

```diff
@@ -19,46 +19,147 @@ public enum ShadowQuality
         SoftShadows,
     }
 
+    /// <summary>
+    /// This controls the size of the shadow map texture.
+    /// </summary>
     public enum ShadowResolution
     {
+        /// <summary>
+        /// Use this for 256x256 shadow resolution.
+        /// </summary>
         _256 = 256,
+
+        /// <summary>
+        /// Use this for 512x512 shadow resolution.
+        /// </summary>
         _512 = 512,
+
+        /// <summary>
+        /// Use this for 1024x1024 shadow resolution.
+        /// </summary>
         _1024 = 1024,
+
+        /// <summary>
+        /// Use this for 2048x2048 shadow resolution.
+        /// </summary>
         _2048 = 2048,
+
+        /// <summary>
+        /// Use this for 4096x4096 shadow resolution.
+        /// </summary>
         _4096 = 4096
     }
 
+    /// <summary>
+    /// This controls the size of the Light Cookie texture.
```

Comment:
**atlas** texture for additional lights (point, spot)

Reply:
Added.